liquibase-core/src/main/java/liquibase/database/core/PostgresDatabase.java

```diff
@@ -296,4 +297,41 @@ private boolean runExistsQuery(String query) throws DatabaseException {
     public CatalogAndSchema.CatalogAndSchemaCase getSchemaAndCatalogCase() {
         return CatalogAndSchema.CatalogAndSchemaCase.LOWER_CASE;
     }
+
+    @Override
+    public String getDatabaseFullVersion() throws DatabaseException {
+        if (getConnection() == null) {
+            throw new DatabaseException("Connection not set. Can not get database version. " +
+                    "Postgres Database wasn't initialized in proper way !");
+        }
+        if (dbFullVersion != null){
+            return dbFullVersion;
+        }
+        final String sqlToGetVersion = "SELECT version()";
+        List<?> result = ExecutorService.getInstance().
+                getExecutor(this).queryForList(new RawSqlStatement(sqlToGetVersion), String.class);
+        if (result != null && !result.isEmpty()){
+            return result.get(0).toString();
+        }
+
+        throw new DatabaseException("Connection set to Postgres type we don't support !");
+    }
+
+    /**
+     * Method to get Postgres DB type
+     * @return Db types
+     * */
+    public DbTypes getDbType() throws DatabaseException {
+        boolean enterpriseDb = getDatabaseFullVersion().toLowerCase().contains("enterprisedb");
+        return enterpriseDb ? DbTypes.EDB : DbTypes.COMMUNITY;
+    }
+
+    /**
+     * Represents Postgres DB types.
+     * Note: As we know COMMUNITY, RDS and AWS AURORA have the same Postgres engine. We use just COMMUNITY for those 3
+     *       If we need we can extend this ENUM in future
+     */
+    public enum DbTypes {
+        EDB, COMMUNITY
```

Comment:
`COMMUNITY` and `AURORA` may have the same engine, but we still need a way to discriminate them (if our backup/restore strategy will be diff, etc)

please move `DbType` enum declaration to top of the class

`getDbType()` can't return EDB as default.
its Datical specific logic and can't be declared on Liquibase level.

Reply:
Liquibase works with EDB for now